sdk/tests/conformance/textures/misc/tex-video-using-tex-unit-non-zero.html

```diff
@@ -0,0 +1,175 @@
+<script id="shader-fs" type="x-shader/x-fragment">
+    precision mediump float;
+
+    uniform sampler2D uTexture0;
+    uniform sampler2D uTexture1;
+
+    varying vec2 vUV;
+
+    void main(void) {
+        vec4 texture0Color = texture2D(uTexture0, vUV);
+        vec4 texture1Color = texture2D(uTexture1, vUV);
+
+        gl_FragColor = mix(texture0Color, texture1Color, 0.5);
+    }
+</script>
+<script id="shader-vs" type="x-shader/x-vertex">
+    attribute vec3 aVertexPosition;
+    attribute vec2 aTextureCoord;
+
+    varying vec2 vUV;
+
+    void main(void) {
+        vUV = aTextureCoord;
+        gl_Position = vec4(aVertexPosition, 1.0);
+    }
+</script>
+
+<script>
+"use strict";
+
+var wtu = WebGLTestUtils;
+description("Regression test for crbug.com/676719");
+
+var gl = wtu.create3DContext("example", {preserveDrawingBuffer: true});
+
+var program = wtu.setupProgram(gl, ["shader-vs", "shader-fs"],
+                               ["aVertexPosition", "aTextureCoord"], [0, 1]);
+
+if (!gl || !program) {
+  testFailed("Test setup failed");
+}
+
+var loc0 = gl.getUniformLocation(program, "uTexture0");
+var loc1 = gl.getUniformLocation(program, "uTexture1");
+if (!loc0 || !loc1) {
+  testFailed("Failed to query uniform locations");
+}
+gl.uniform1i(loc0, 0);
+gl.uniform1i(loc1, 1);
+
+wtu.setupUnitQuad(gl, 0, 1);
+
+// Load textures
+var imageElement = document.getElementById("src-image");
+var videoElement = document.getElementById("src-video");
+
+// Need to wait for video to be ready before continuing
+videoElement.addEventListener('canplay', runTest);
+
+function runTest() {
+  // Prevent further calls to oncanplay
+  videoElement.removeEventListener('canplay', runTest);
```

Comment:
If the video actually needs to play at all, this would be a good place to pause it.